.github/workflows/ci.yml

```diff
@@ -130,12 +130,12 @@ jobs:
           make ci-test
 
       - name: Coveralls
-        run: vendor/bin/php-coveralls -v --coverage_clover='clover.xml' --json_path='coveralls-upload.json'
-        if: env.COVERALLS_REPO_TOKEN && matrix.php-version == '8.2' && matrix.env == 'AUTOLOAD=1'
-        env:
-          COVERALLS_RUN_LOCALLY: 1
-          COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
```

Comment:
We can probably remove the `COVERALLS_REPO_TOKEN` environment variable (if present) from https://github.com/stripe/stripe-php/settings/variables/actions.